src/leap/bitmask/services/mail/smtpbootstrapper.py

```diff
@@ -99,9 +102,21 @@ def _download_config_and_cert(self):
                 check_and_fix_urw_only(client_cert_path)
                 return
 
-            download_client_cert(self._provider_config,
-                                 client_cert_path,
-                                 self._session)
+            try:
+                download_client_cert(self._provider_config,
+                                     client_cert_path,
+                                     self._session, kind="smtp")
+            except HTTPError:
```

Comment:
you're right. I quickly looked and saw that in the place where the exception is catched we don't have the return code, *but* it is in the exception message.